### ISSUE TITLE
Bootstrap progressbar missing 

### ISSUE THREAD

**Issue by anjandas0211** [yiister/yii2-gentelella#30]
After downloading template when going to setup then it shows bellow error
vendor\bower/gentelella/vendors/bootstrap-progressbar

**Comment by zhukovra** [comment]
Please provide more information how you install `yii2-gentella`. Did you read **Installation** in README.md?

**Comment by anjandas0211** [comment]
yeah, read that and I replace my main layout file with yii2-gentelella main template file code

**Comment by zhukovra** [comment]
Do you have `fxp/composer-asset-plugin` installed globally?

**Comment by anjandas0211** [comment]
Yes. In my bower asset vendor file this folder existed but it show this error

**Comment by zhukovra** [comment]
Please attach more detailed error.

**Comment by fps01** [comment]
@anjandas0211 Hello!
Could you tell me more information?
P. S. The  `vendor\bower/gentelella/vendors/bootstrap-progressbar` path has different slashes. What OS do you use?
P. P. S. Do you have `vendor\bower` or `vendor\bower-asset` directory?

**Comment by raphaelramosds** [comment]
On your application composer.json, just rewrite the bower-asset/gentella version to

```
"bower-asset/gentelella": "~1.3"
```

**Reason:** ThemeAsset on the latest version of yiister/yii2-gentelella will try to resolve yiister\gentelella\assets\BootstrapProgressbar, that does not exist on bower-asset/gentelella latest releases (~2.0)

**Comment by KabataMacharia** [comment]
> On your application composer.json, just rewrite the bower-asset/gentella version to
> 
> ```
> "bower-asset/gentelella": "~1.3"
> ```
> 
> **Reason:** ThemeAsset on the latest version of yiister/yii2-gentelella will try to resolve yiister\gentelella\assets\BootstrapProgressbar, that does not exist on bower-asset/gentelella latest releases (~2.0)

Added this to my composer json file and it worked
